feat(microservices): enhance Modbus protocol properties documentation

sansmoraxz opened the pull request:
Linked to https://github.com/edgexfoundry/device-modbus-go/pull/688

<!-- Add additional detailed description of need for change if no related issue -->

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-docs/blob/main/.github/Contributing.md 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Changes have been rendered and validated locally using mkdocs-material (see edgex-docs README)

sansmoraxz committed:
feat(microservices): enhance Modbus protocol properties documentation

Signed-off-by: Souyama <souyamadebnath@gmail.com>

sansmoraxz commented:
Also fixed some indentation issues due to which mardown parser was not rendering code blocks correctly

cloudxxx8 commented on docs_src/microservices/device/services/device-modbus/ProtocolProperties.md:
````diff
@@ -9,75 +9,82 @@ Device Modbus defines the following Protocol Properties for Modbus-TCP, Modbus-R
+}}
+```
+
+> **_NOTE:_** About the timeout values: It can be a number (floating point) in which case it is interpreted as seconds. Or it can be a duration string such as "500ms", "2s" or "1.5m" representing milliseconds, seconds or minutes respectively.
````
please move this notes to each timeout value.
although it's a little redundant, users can get this easier.

For example:
```
| Timeout                  | The timeout when connecting or reading Device Modbus Service to device-modbus. It can be a number (floating point) in which case it is interpreted as seconds, or it can be a duration string such as "500ms", "2s" or "1.5m" representing milliseconds, seconds or minutes respectively.                                                                    |
| IdleTimeout              | Idle timeout to close the connection (use 0 to dial for each request and negative value to never close) - default 0. It can be a number (floating point) in which case it is interpreted as seconds, or it can be a duration string such as "500ms", "2s" or "1.5m" representing milliseconds, seconds or minutes respectively.                              |
| LinkRecoveryTimeout      | Recovery timeout if tcp communication misbehaves - default 0. It can be a number (floating point) in which case it is interpreted as seconds, or it can be a duration string such as "500ms", "2s" or "1.5m" representing milliseconds, seconds or minutes respectively.                                                                                     |
| ProtocolRecoveryTimeout  | Recovery timeout if the protocol is malformed, e.g. wrong transaction ID - default 0. It can be a number (floating point) in which case it is interpreted as seconds, or it can be a duration string such as "500ms", "2s" or "1.5m" representing milliseconds, seconds or minutes respectively.                                                             |
| ConnectDelay             | Silent period after successful connection - default 0. It can be a number (floating point) in which case it is interpreted as seconds, or it can be a duration string such as "500ms", "2s" or "1.5m" representing milliseconds, seconds or minutes respectively.                                                                                            |
```

cloudxxx8 replied:
```suggestion
> **_NOTE:_** About the timeout values: It can be a number (floating point) in which case it is interpreted as seconds, or it can be a duration string such as "500ms", "2s" or "1.5m" representing milliseconds, seconds or minutes respectively.
```

sansmoraxz committed:
fix: lint

Update docs_src/mcroservices/device/services/device-modbus/ProtocolProperties.md

Co-authored-by: Cloud Tsai <cloudxxx8@gmail.com>
Signed-off-by: Souyama <souyamadebnath@gmail.com>

sansmoraxz committed:
fix: improve timeout description

Signed-off-by: Souyama <souyamadebnath@gmail.com>

cloudxxx8 reviewed (approved):
LGTM
